### ISSUE TITLE
Not remebering

### ISSUE THREAD

**Issue by ghost** [chetan51/sidestep#63]
Hello,

Everytime I open Sidestep, it does not automatically connect to the set SSH profile. I have to connect within the preferences, then try connecting, then re-enter the password (which doesn't save in the Keychain btw) and then it finally connects. What's the problem, or is it me?

**Comment by DenBeke** [comment]
I have also problems remembering the password.  
If I check the 'remember checkbox', Sidestep will always have a `authorization failure` next time I try to connect. When trying to connect again after the failure, Sidestep will again ask for my SSH password. In short I'm not able to store my password... (OS X 10.9)

**Comment by RwwL** [comment]
I'm experiencing this too, on multiple computers; both on Mavericks as well.

**Comment by RwwL** [comment]
I just set up passwordless ssh access on my proxy server to get around the whole Keychain issue and it seems to have worked, I can connect every time now.

**Comment by jr00n** [comment]
@RwwL what do you mean by passwordless? ssh trusted host?

**Comment by RwwL** [comment]
@jr00n Yes. I think there are multiple ways to set that up... I put my public key into .ssh/authorized_keys in my user folder on the server.

**Comment by quicoto** [comment]
Same here, password not being remembered.

**Comment by thalessr** [comment]
I have this same problem as well

**Comment by louiswalch** [comment]
Same, please fix. So frustrating.

**Comment by quicoto** [comment]
+1
